dotCMS/src/main/java/com/dotmarketing/cms/myaccount/action/MyInterestsAction.java

```diff
@@ -24,7 +25,8 @@
 import com.dotmarketing.util.WebKeys;
 import com.liferay.portal.model.User;
 
-public class MyInterestsAction extends DispatchAction {
+@Deprecated
+public class MyInterestsAction extends SecureAction {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)